src/client/keys.rs

```diff
@@ -0,0 +1,107 @@
+use serde::{Deserialize, Serialize};
+
+use super::Client;
+use crate::transport::HttpLowLevel;
+
+/// To interact with the Keys API.
+pub struct ClientKeys<'a, T> {
+    pub(super) client: Client<'a, T>,
+}
+
+impl<'a, T> ClientKeys<'a, T>
+where
+    T: HttpLowLevel,
+{
+    /// Create an API Key.
+    pub async fn create(
+        &self,
+        actions: Vec<String>,
```

Comment:
I think that this should be an enum type, or T: AsRef<str>,  and create constants for the values defined in https://typesense.org/docs/0.19.0/api/api-keys.html#sample-actions